### ISSUE TITLE
PHP 8.1 : Deprecated Functionality: curl_multi_exec(): Passing null to parameter

### ISSUE THREAD

**Issue by Dnd-Juver** [wl-online-payments-direct/sdk-php#7]
Hi,

We have this error on PHP 8.1 ( wl-online-payments-direct/sdk-php versions : 4.5.0)

```
Deprecated Functionality: curl_multi_exec(): Passing null to parameter #2 ($still_running) of type int is deprecated 

vendor/wl-online-payments-direct/sdk-php/lib/OnlinePayments/Sdk/DefaultConnection.php on line 191
vendor/worldline/module-magento-payment-core/Model/Config/ConnectionTest/FromAjaxRequest.php:59
vendor/worldline/module-magento-payment-core/Controller/Adminhtml/System/Config/TestConnection.php:45
```

PHP Doc :
[https://www.php.net/manual/en/function.curl-multi-exec.php](https://www.php.net/manual/en/function.curl-multi-exec.php) `curl_multi_exec(CurlMultiHandle $multi_handle, int &$still_running): int`

File :
https://github.com/wl-online-payments-direct/sdk-php/blob/master/lib/OnlinePayments/Sdk/DefaultConnection.php#L191

Regards,
Julien

**Comment by worldline-direct-support-team** [comment]
Hi Julien,

Thank you for the data, we will look into this.

Kind regards,
Worldline Direct support team

**Comment by Dnd-Juver** [comment]
Hi,

The error seems to occur on the PHP 8.1.7 version.
We do not have an error on PHP Version 8.1.13

**Comment by worldline-direct-support-team** [comment]
Hello Dnd-Juver,

A new version of the SDK was released that is compatible with PHP version 8.0 and up. Including version 5.0.0 of the SDK should resolve your issues.

Kind regards,
Worldline Support
